### PR TITLE
Portfolio Mobile Menu

### PR DESCRIPTION
**Pull request by Yerimah**
## Mobile Menu Interactiveness

In this branch, we implemented the following:
- Edited the mobile menu in the markup file 
- Modified the style in the stylesheet file
- Added an index.js file and made the mobile menu interactive
- Added Eslint for javascript

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  toggleMenu.style.display = 'flex';
+  toggleMenu.style.top = '0';
+}
+
+function close() {
+  toggleMenu.style.top = '-100%';
+}
+
+hamburger.addEventListener('click', show);
+closetog.addEventListener('click', close);
```
Comment (elyor-doniyorov) [inline index.js:15]: - [ ] Good job 👏  You implemented all features in the requirements except only one. Kindly add a feature that `when the user clicks (or taps) any option of the mobile menu, the mobile menu disappears`.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi Team,👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Yerimah) [reply index.js:15]: Feature implemented

**Review by elyor-doniyorov — APPROVED** [review]
Hi @Yerimah and @BertrandConxy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**